src/components/Button/src/Button.vue

```diff
@@ -130,21 +151,44 @@ export default {
 			default: undefined,
 		},
 		/**
-		 * Background color of button
+		 * MText pattern in button label. Pattern's size only applies on (`size: undefined`) buttons.
+		 */
+		textPattern: {
+			type: String,
+			default: undefined,
```

Comment:
the default for `textPattern` within MButton is `undefined`, and when it is `undefined` it's the same as deferring the default value to the MText component, in which case it will always resolve to `paragraph`, which functionally means the default for MButton also becomes `paragraph`, however if we check MButton's CSS styles we'll see that the default text pattern for buttons, and for all interactive clickable components, is suppose to be `label`, so to maintain that behavior we need to switch this `undefined` to `'label'`. And if `textPattern` is eventually made into a themeable prop we'd have to add `textPattern: 'label'` to MButton's default theme.